src/main/java/org/ow2/proactive/catalogclient/service/SpringConfig.java

```diff
@@ -28,14 +28,13 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
 import org.springframework.context.annotation.PropertySource;
 import org.springframework.context.annotation.PropertySources;
 import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
 
 
-@Configuration
 @ComponentScan(basePackages = { "org.ow2.proactive.catalogclient.*" })
-@PropertySources({ @PropertySource("classpath:catalogclient/application.properties") })
 public class SpringConfig {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1118)